packages/grafana-ui/src/components/Collapse/CollapsableSection.tsx

```diff
@@ -49,8 +52,11 @@ export const CollapsableSection = ({
     e.preventDefault();
     e.stopPropagation();
 
-    onToggle?.(!open);
-    toggleOpen(!open);
+    if (isControlled) {
+      onToggle?.(!isOpen);
+    } else {
+      toggleInternalOpenState(!internalOpenState);
+    }
```

Comment:
```suggestion
    if (!isControlled) {
      toggleInternalOpenState(!internalOpenState);
    }  
    onToggle?.(!isOpen);
```
We'd probably call `onToggle` as before, to not break compatibility.